recipes/virtual-libjpeg/all/conanfile.py

```diff
@@ -0,0 +1,12 @@
+from conans import ConanFile, tools
+
+
+class VirtualjpegConan(ConanFile):
+    name = "virtual-libjpeg"
+    description = "Virtual package to choose jpeg encoder/decoder implementation in final package."
+    url = "https://github.com/conan-io/conan-center-index"
+    options = {"implementation": "ANY"}
+    default_options = {"implementation": "libjpeg-turbo/2.0.5"}
+
+    def requirements(self):
+        self.requires(str(self.options.implementation))
```

Comment:
- Not a fan of putting the name+version as a value.
This is not an easy format to parse by consumers.
- What default should we choose?
I though most recipes used `libjpeg` by default and `libjpeg-turbo` is patched in.
- Also, should the CI be modified to check, when a recipe requires this virtual package, whether the recipe actually is able to build with both libjpeg implementations?
- Is there a better sounding synomym for *implementation*?
```suggestion
    options = {
        "implementation": ["libjpeg", "libjpeg-turbo"],
    }
    default_options = {
        "implementation": "libjpeg-turbo",
    }

    def requirements(self):
        if self.options.implementation == "libjpeg":
            self.requires("libjpeg/9d")
        elif self.options.implementation == "libjpeg-turbo":
            self.requires("libjpeg-turbo/2.0.5")
        self.requires(str(self.options.implementation))
```